gulp/util/create-config.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2015 Palantir Technologies, Inc. All rights reserved.
+ */
+"use strict";
+
+module.exports = function createConfig (project) {
```

Comment:
at the very least, filename should have some mention of "karma" so we know what it's for. like its sibling, `webpack-config.js`.

but i had no problem with this config living in karma.js directly as it's only needed there.